src/main/java/com/codeborne/selenide/appium/AppiumClickOptions.java

```diff
@@ -0,0 +1,50 @@
+package com.codeborne.selenide.appium;
+
+import com.codeborne.selenide.ClickMethod;
+import com.codeborne.selenide.ClickOptions;
+import com.codeborne.selenide.appium.commands.AppiumClickMethod;
+
+public class AppiumClickOptions extends ClickOptions {
+
+  private final AppiumClickMethod appiumClickMethod;
+  private final int xOffset;
```

Comment:
@amuthansakthivel I guess you don't need `offsetX` and `offsetY` in this class?
Parent class `ClickOptions` already has these fields.

Reply:
@asolntsev Those are private. It has to be made as protected in order for me to use them.

Reply:
@amuthansakthivel No, the fields are private, BUT you can still
1. pass these values to constructor (it's protected), and
2. get these values using getters `offsetX()`, `offsetY()` (they are also protected).

Reply:
oops my bad. How did I missed it. Fixed it now